src/main/java/org/myrobotlab/service/Tracking.java

```diff
@@ -303,6 +305,68 @@ public ServiceConfig getConfig() {
     return config;
   }
 
+  static public Map<String, ServiceConfig> getDefaultConfig() {

```

Comment:
This method should be removed...  you have getDefault() below..   don't put default configs in 2 places on this service.